zoninator.php

```diff
@@ -1513,15 +750,7 @@ function _get_zone_page_url( $args = array() ) {
 		return $url;
 	}
 
-	function _validate_date_filter( $date ) {
-		return preg_match( '/([0-9]{4})-([0-9]{2})-([0-9]{2})/', $date );
-	}
-
-	function _validate_category_filter( $cat ) {
-		return $cat && get_term_by( 'id', $cat, 'category' );
-	}
-
-	function _get_value_or_default( $var, $object, $default = '', $sanitize_callback = '' ) {
+	private function _get_value_or_default( $var, $object, $default = '', $sanitize_callback = '' ) {
```

Comment:
Will be changed to public. Alternatively, we could do something like this to eventually deprecate their public use

[deprecations.diff.zip](https://github.com/Automattic/zoninator/files/86654/deprecations.diff.zip)